benchmark/buffers/buffer-iterate.js

```diff
@@ -31,15 +31,15 @@ function main({ size, type, method, n }) {
 function benchFor(buffer, n) {
   for (let k = 0; k < n; k++) {
     for (let i = 0; i < buffer.length; i++) {
-      assert(buffer[i] === 0);
+      assert.ok(buffer[i] === 0);
```

Comment:
It might make sense to switch to `strictEqual` instead
```suggestion
      assert.strictEqual(buffer[i], 0);
```